com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -311,12 +311,55 @@ void DrawElementCallback(Rect rect, int index, bool isActive, bool isFocused)
             }
         }
 
+        // Modified version of StageHandle.FindComponentsOfType<T>()
+        // This version more closely represents unity object referencing restrictions.
+        // I added these restrictions:
+        // - Can not reference scene object outside scene
+        // - Can not reference cross scenes
+        // - Can reference child objects if it is prefab
+        Camera[] FindCamerasToReference(GameObject gameObject)
+        {
+            var scene = gameObject.scene;
+
+            var notInScene = EditorUtility.IsPersistent(camera) && !scene.IsValid();
```

Comment:
Could we have this as a positive variable `inScene` like the others. I find that more readable if there ever is negations and it's consistent with the other bools.

Reply:
k done it